frontend/src/fetchers/ssdi.js

```diff
@@ -95,9 +95,11 @@ export const getSelfServiceData = async (filterName, filters, dataSetSelection =
   const url = getSelfServiceUrl(filterName, filters);
 
   const urlToUse = url + dataSetSelection.map((s) => `&dataSetSelection[]=${s}`).join('');
-  const response = await get(urlToUse);
-  if (!response.ok) {
-    throw new Error('Error fetching self service data');
-  }
-  return response.json();
+
+  return get(urlToUse).then((response) => {
+    if (!response.ok) {
+      throw new Error('Error fetching self service data');
+    }
+    return response.json();
+  });
```

Comment:
what is the reason for this change? this code is functionally identical

Reply:
it returns a promise now, it did not before

Reply:
so now the requests are actually performed in parallel

Reply:
It doesn't matter to me if you change it back or not I was just trying to understand why we are breaking convention here

Reply:
but it was still waiting for get(urlToUse) to comeback which is what was taking the time before returning the promise